package.json

```diff
@@ -13,13 +13,16 @@
   },
   "dependencies": {
```

Comment:
Please bump up app version.

Reply:
Will do, was waiting to see what it was currently. 
Is changing daily 😆

Reply:
> Looks good to me but please merge changes from master. There are some little changes I did for CompanyProvisions.vue

Will do, thanks the heads up Andre! 👍

Reply:
Bumped to `0.5.16` 🤞 😆